tutor/src/components/tutor-popover.cjsx

```diff
@@ -51,7 +51,6 @@ TutorPopover = React.createClass
     content.querySelectorAll('img')
 
   imageLoaded: (iter) ->
-    return unless @isMounted()
```

Comment:
Yeah, this component doesn't really have any listeners, it's purpose is to load and display a popover when the mouse moves over an element.

Since the `imagesLoaded` call originates from mouse `onEnter` , it's guaranteed to only be called when mounted, otherwise it wouldn't be receiving events.

And if i've mis-read the way the code somehow, the worst that happens is that we call `setState` on an unmounted component and React logs a warning.